docs/migration.md

```diff
@@ -0,0 +1,24 @@
+# Migrating to OpenTelemetry .NET
+
+Follow these steps if you want to migrate from this distribution to the
+upstream OpenTelemetry .NET project:
+
+- Replace all environment variables or system properties with the "grafana"
+prefix as explained[here](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/#push-directly-from-applications-using-the-opentelemetry-sdks).
+- Install and activate the OpenTelemetry SDK,
```

Comment:
We should document the need to set up an OTLP exporter. The upstream getting started docs don't document setting up an OTLP exporter.

Without calling `AddOtlpExporter`, the related environment variables will not be considered.